eng/tools/typespec-validation/tsconfig.json

```diff
@@ -1,11 +1,6 @@
 {
-  "extends": "../../../tsconfig.json",
+  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "target": "ES6",
-    "module": "Node16",
     "outDir": "./dist",
```

Comment:
Cannot be set in base config because:

> Properties with relative paths found in the configuration file ... will be resolved relative to the configuration file they originated in.

https://www.typescriptlang.org/tsconfig/#extends